smarc_bringup/launch/auv_model.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
+
+	<!-- AUV model -->
+	<arg name="namespace" default="lolo_auv"/>
+	<!-- Index of the AUV to be launched -->
+	<arg name="number" default="3"/>
```

Comment:
Small nitpick: Why can't we just have `type` with the model and `<arg name="namespace" default="$(arg type)"/>`. This would enable us to avoid having `lolo_auv_3` as the default namespace.

Reply:
sure, np.
Can you make the changes?